python_pybind_test.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
@mrpollo @julianoes This is functionally the same example as for C++. We're going to want it as an example, in a sensible folder structure that identifies the test programming language and the test name (allowing us to have multi-file tests). Ideally this should match the C++ test structure.

Reply:
agreed, I added a commit to move it